packages/javascript_common/src/lib/sf_host.ts

```diff
@@ -21,6 +23,7 @@ function strace<A extends unknown[], R>(name: string, fn: Fn<A, R>, asyncify: As
     return result;
   }
 }
+// @ts-ignore ignore unused function
```

Comment:
I left these thinking it can come in handy in two cases:
1. as a documentation of how to work with the ABI from TypeScript
2. in case you are debugging your code by monkey-patching it, then these can come in handy since they will be in the dist already. However I did not quite realize that bundlers will probably notice these are unused and remove them.

I think we can safely remove them.